### DISCUSSION TITLE
is there any plan to move these tutorials to main Avalonia documentation site?

### DISCUSSION THREAD

**Discussion by nsrahmad** [AvaloniaUI/Avalonia.Samples#63] [Ideas]

**Comment by timunie** [comment]
No, there are no such plans. They will stay in this repo. The other way around is planned, moving the tutortials "ToDo-List" and "Music-Store" into this repo with working code to download. 

However, we have links to these samples where useful. If you think there are other places in the docs where we should link a sample, let me know.

**Reply by nsrahmad** [reply]
This is interesting. What's the reason to keep the tutorials in the Samples repo?
